app/uk/gov/hmrc/gform/validation/ComponentValidator.scala

```diff
@@ -123,7 +123,11 @@ object ComponentValidator {
 
     def lookupError: ValidatedType[Unit] = {
       val vars: List[String] = lookupLabel.label :: Nil
-      validationFailure(fieldValue, genericErrorLookup, Some(vars))
+
+      val shortNameForErrorMessage =
+        SmartString(LocalisedString(Map(LangADT.En -> "a value", LangADT.Cy -> " werth")), Nil)
```

Comment:
Why don't put `"a value"` directly to the message?

Reply:
good point!